### ISSUE TITLE
Textured image in the paper teaser

### ISSUE THREAD

**Issue by xyyeah** [qianlim/SCALE#10]
Thanks for the exciting job. How to get the textured image in the paper teaser?

**Comment by qianlim** [comment]
Hi, if you mean the colored ('with texture') point set in the teaser, when the GT point color is available, we train a model that predicts the normals and point colors using two separate network branches (see supmat). If you mean the 'neural rendered' results, we used [SMPLpix](https://github.com/sergeyprokudin/smplpix); maybe [this discussion](https://github.com/sergeyprokudin/smplpix/issues/11) is also helpful to you.

**Comment by xyyeah** [comment]
Thanks for your quick reply. I mean how to get the colored point set. I downloaded the cape dataset, but it doesn’t seem to contain images or textures. Or am i missing something?

**Comment by qianlim** [comment]
Right, the CAPE meshes contain only geometry but no texture. As the training data of SCALE is actually sampled point clouds from the CAPE mesh surfaces, what we did is that we load the corresponding raw scan for each CAPE mesh, and transfer the color to the sampled points. You can request the raw scans of some subjects and sequences from CAPE, see their website for more details.

**Comment by xyyeah** [comment]
![image](https://user-images.githubusercontent.com/90754199/146887567-d5fac0d7-fe4e-4331-8297-cfda3ad83786.png)
Still cannot find colors from the raw scans, where can i find it ?

**Comment by qianlim** [comment]
By the way I forgot to mention that the raw scans are by default not colored either. You can specify that you need colored scans when requestion.
Sorry I deleted your most recent post. Please do not share any format (including renderings, screenshots) of the raw scans in public.
